.env.example

```diff
@@ -0,0 +1,15 @@
+# Domain options
+# DEV: 
+# core: https://dev.getunblock.com 
+# fiat-connect: https://dev.getunblock.com/fiat-connect/
+# STAGING: 
+# core: https://sandbox.getunblock.com
+# fiat-connect: https://sandbox.getunblock.com/fiat-connect
+# Prod: 
```

Comment:
Capital letter as well

Reply:
Yeah not sure why I did not put it capital as well 👍